### ISSUE TITLE
Issue with loading fine-tuned BLEURT

### ISSUE THREAD

**Issue by vaiibhavgupta** [google-research/bleurt#56]
I am unable to load the fine-tuned model. It is falling back and loading the base model.

How I am loading the model - 
`model = evaluate.load("bleurt", module_type="metric", checkpoint="/finetuned_bleurt/export/bleurt_best/1689684085/")
`

I also tried the following, but same issue - 
`bleurt_model = evaluate.load("bleurt", module_type="metric", checkpoint="/finetuned_bleurt_base_128/")
`

What am I doing wrong here?
